src/ServiceControl.Audit/Auditing/BodyStorage/BodyStorageFeature.cs

```diff
@@ -1,6 +1,5 @@
 ﻿namespace ServiceControl.Audit.Auditing.BodyStorage
 {
-    using System;
```

Comment:
I guess this change is not strictly necessary. I'm good though to take it in

Reply:
Tidying up as I go along